client/src/components/Tabs/TabRenderer.tsx

```diff
@@ -21,9 +21,9 @@ const TabRenderer: FC<Props> = ({ tab, topic }) => {
     default:
       return (
         <Alert severity="error">
-          Invalid tab: '
+          Invalid tab: &apos;
```

Comment:
Could the apos and {tab} go on the same line?

Reply:
I tried doing that and eslint complains that they should be on separate lines